packages/engine-formula/src/engine/reference-object/table-reference-object.ts

```diff
@@ -16,7 +16,7 @@
 
 import type { ISuperTable } from '../../basics/common';
 import { TableOptionType } from '../../basics/common';
-import { $SUPER_TABLE_COLUMN_REGEX } from '../../basics/regex';
+import { $SUPER_TABLE_COLUMN_REGEX_PRECOMPILING } from '../../basics/regex';
```

Comment:
why this variable start with "$", it there some rule I don't know?

Reply:
fixed